operator/v1/types_network.go

```diff
@@ -788,3 +814,38 @@ const (
 	// between pods on the cluster network.
 	IPsecModeFull IPsecMode = "Full"
 )
+
+// +kubebuilder:validation:Enum:="";"Enabled";"Disabled"
+type RouteAdvertisementsEnablement string
+
+var (
+	// RouteAdvertisementsEnabled enables route advertisements for ovn-kubernetes
+	RouteAdvertisementsEnabled RouteAdvertisementsEnablement = "Enabled"
+	// RouteAdvertisementsDisabled disables route advertisements for ovn-kubernetes
+	RouteAdvertisementsDisabled RouteAdvertisementsEnablement = "Disabled"
+)
+
+// RoutingCapabilitiesProvider is a component providing routing capabilities.
+// +kubebuilder:validation:Enum=FRR-k8s
+type RoutingCapabilitiesProvider string
+
+const (
+	// RoutingCapabilitiesProviderFRRk8s determines FRR-k8s is providing
+	// advanced routing capabilities.
+	RoutingCapabilitiesProviderFRRk8s RoutingCapabilitiesProvider = "FRR-k8s"
```

Comment:
FRR-k8s is not conformant to the PascalCase requirement for enum values, what about FRRKubernetes or FRRDaemon, or just FRRouting?

Reply:
Unusre if it is because of the dash or lower case `k`. Can we do `FRR-K8s`?

Reply:
This is a proper noun so we can only change it so far until it just stops referring to the thing we want to refer to. If `FRR-K8s` does not work, would `FRRk8s` or `FRRK8s` work?

Reply:
`FRRK8s` would be an appropriate PascalCase yes